src/main/config/config_eeprom.c

```diff
@@ -158,6 +159,15 @@ uint16_t getEEPROMConfigSize(void)
     return eepromConfigSize;
 }
 
+size_t getEEPROMStorageSize(void)
+{
+#ifdef EEPROM_IN_RAM
+    return EEPROM_SIZE;
```

Comment:
Where is `EEPROM_SIZE` defined? Shouldn't `EEPROM_IN_RAM` be handled in the same way as `EEPROM_IN_RAM`?

Reply:
I'll move the code change from #8379 that sets a default for EEPROM_SIZE into this PR.  For the initial implementation of this the target defined it.

Reply:
done